### PR TITLE
Load Bit api

### PR DESCRIPTION
**Pull request by ranm8**
## Proposed Changes

- we need to discuss how to use the loadBit API and how to use from core aspects and how dependency strategy.
- discuss testing and test-kits.

### CODE REVIEW

File: e2e/harmony/load-bit.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { loadBit } from '@teambit/bit';
+import { ScopeAspect, ScopeMain } from '@teambit/scope';
+import { Workspace, WorkspaceAspect } from '@teambit/workspace';
+import { expect } from 'chai';
+import Helper from '../../src/e2e-helper/e2e-helper';
+import { HARMONY_FEATURE } from '../../src/api/consumer/lib/feature-toggle';
+
+describe.only('loadBit()', function () {
```
Comment (davidfirst) [inline e2e/harmony/load-bit.e2e.ts:8]: @ranm8 , don't forget to remove the "only".

File: scopes/react/react/react.env.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ import previewConfigFactory from './webpack/webpack.preview.config';
 import eslintConfig from './eslint/eslintrc';
 
 export const AspectEnvType = 'react';
-const jest = require('jest');
+const jestM = require('jest');
```
Comment (davidfirst) [inline scopes/react/react/react.env.ts:26]: @ranm8 , any reason to keep this change?

File: workspace.jsonc [hunk 3]
```diff
@@ -8,7 +8,7 @@
     // "vendorDirectory": "vendor",
   },
   "teambit.dependencies/dependency-resolver": {
-    "packageManager": "teambit.dependencies/pnpm",
+    "packageManager": "teambit.dependencies/yarn",
```
Comment (davidfirst) [inline workspace.jsonc:11]: @ranm8 , revert this change, otherwise, many tests will fail.

File: scopes/react/react/react.env.ts (hunk 2 above)
Comment (ranm8) [reply scopes/react/react/react.env.ts:26]: Better keep it, seems like the Jest transpiler adds const jest = require('jest') for workspace component testing.

File: e2e/harmony/load-bit.e2e.ts (hunk 1 above)
Comment (ranm8) [reply e2e/harmony/load-bit.e2e.ts:8]: thanks!

File: workspace.jsonc (hunk 3 above)
Comment (ranm8) [reply workspace.jsonc:11]: Thanks!
